packages/postcss-minify-gradients/src/index.js

```diff
@@ -162,7 +162,7 @@ function optimise(decl) {
           color = color.toLowerCase();
 
           const colorStop =
-            stop || stop === 0
```

Comment:
`stop` is either `undefined` or contains a node value which in general is a string, so it should never be the number `0`, unless someone assigned the number `0` to the node value by mistake. Anyway, I think the intent was to check whether `stop` had been initialized, so I have made it explicit.